ha/setup/ha_setup.py

```diff
@@ -872,7 +872,7 @@ def process(self):
 
             self._remove_logs(older_logs)
         except Exception as e:
-            Log.error(f"Cluster reset command failed. Error: {e}")
+            sys.stderr.write(f"Cluster reset command failed. {traceback.format_exc()}, Error: {e}\n")
```

Comment:
Please confirm that when exception is received, this traceback gets called and it displays correct and all the expected data 
You can force an exception in your test and confirm the same

Reply:
Tested & confirmed. Added test details in PR